src/main/java/com/hivemq/cli/commands/AbstractCommonFlags.java

```diff
@@ -62,9 +62,9 @@ private void setPasswordFromEnv(final @NotNull ByteBuffer passwordEnvironmentVar
     private @Nullable Integer keepAlive;
 
     @SuppressWarnings("unused")
-    @CommandLine.Option(names = {"-c", "--cleanStart"}, negatable = true,
+    @CommandLine.Option(names = {"-c", "--cleanStart"}, negatable = true, defaultValue = "true",
```

Comment:
Why was this defaultVaule added?

Reply:
The description defines true and as this is now a primitive Option the default value would be false.